### DISCUSSION TITLE
Calendar - Popup will automatically turn off after the user selects a specific date

### DISCUSSION THREAD

**Discussion by PhuongKieu97** [primefaces/community#3068] [PrimeReact]
![image](https://github.com/user-attachments/assets/45ad70e1-4889-42c3-86d2-1604b0ad0437)
I want to customize the function like the user must click the confirm button to complete the selection. Do you know how I can do that?

**Comment by xAl3xFx** [comment]
Hi,
https://stackblitz.com/edit/9ds66b?file=src%2FApp.jsx

This is the closest I could get to doing what you are trying. Not very good solution but it works :)
